lib/matplotlib/axes/_axes.py

```diff
@@ -5239,21 +5237,27 @@ def pcolor(self, *args, **kwargs):
             An mpl color or sequence of colors will set the edge color
         alpha : scalar, optional, default: None
             The alpha blending value, between 0 (transparent) and 1 (opaque).
-        snap : bool, optional, default: TODO
+        snap : bool, optional, default: False
             Whether to snap the mesh to pixel boundaries.
 
+        Returns
+        -------
+        collection : `matplotlib.collections.Collection`
 
-        Return value is a :class:`matplotlib.collections.Collection`
-        instance.
-
-        .. _axes-pcolor-grid-orientation:
+        See Also
+        --------
+        :func:`~matplotlib.pyplot.pcolormesh`
+            For an explanation of the differences between
+            pcolor and pcolormesh.
 
+        Notes
+        -----
         The grid orientation follows the MATLAB convention: an
-        array *C* with shape (*nrows*, *ncolumns*) is plotted with
-        the column number as *X* and the row number as *Y*, increasing
+        array `C` with shape (`nrows`, `ncolumns`) is plotted with
```

Comment:
Single backticks are references, but they don't work for arguments, unfortunately.